FBSimulatorControlTests/Tests/FBSimulatorControlSimulatorLaunchTests.m

```diff
@@ -137,3 +147,49 @@ - (void)testLaunchesMultipleSimulatorsConcurrently
 }
 
 @end
+
+@implementation FBSimulatorControlSimulatorLaunchTests_DefaultSet
+
+- (NSString *)deviceSetPath
+{
+  return nil;
+}
+
+- (void)testLaunchesSingleSimulator
+{
+  [self doTestLaunchesSingleSimulator];
+}
+
+- (void)testLaunchesMultipleSimulatorsConcurrently
+{
+  [self doTestLaunchesMultipleSimulatorsConcurrently];
+}
+
+@end
+
+@implementation FBSimulatorControlSimulatorLaunchTests_CustomSet
+
+- (NSString *)deviceSetPath
+{
+  return [NSTemporaryDirectory() stringByAppendingPathComponent:@"FBSimulatorControlSimulatorLaunchTests_CustomSet"];
+}
+
+- (void)testLaunchesSingleSimulator
+{
+  if (!FBSimulatorControlStaticConfiguration.supportsCustomDeviceSets) {
+    NSLog(@"-[%@ %@] can't run as Custom Device Sets are not supported for this version of Xcode", NSStringFromClass(self.class), NSStringFromSelector(_cmd));
```

Comment:
Shouldn't we log this when the method is actually attempted to be used on non-supported platform rather then in tests?